business/namespaces.go

```diff
@@ -129,12 +128,133 @@ func (in *NamespaceService) GetNamespaces(ctx context.Context) ([]models.Namespa
 		labelSelectorExcludeValue = excludeLabelList[1]
 	}
 
+	namespaces := []models.Namespace{}
+	// Merge namespaces from different clusters
+	// Namespace sameness
+	nsmap := make(map[string]models.Namespace)
+
+	if clientFactory == nil {
+		var err error
+		clientFactory, err = kubernetes.GetClientFactory()
+		if err != nil {
+			log.Errorf("Error getting client factory: %v", err.Error())
+			return nil, fmt.Errorf("Error getting client factory")
+		}
+	}
+
+	clusterNames := clientFactory.GetClusterNames()
+
+	wg := &sync.WaitGroup{}
+	type result struct {
+		cluster string
+		ns      []models.Namespace
+		err     error
+	}
+	resultsCh := make(chan result)
+
+	go func(ctx context.Context) {
```

Comment:
is the ctx used in function?

Reply:
Thanks, @hhovsepy 
I've added the change